Detox 20 (#3285)
wix/Detox: docs/introduction/partials/_getting-started-other.mdx

:::caution Note

At the moment, Detox does not support non-React Native apps on **Android** due to the blocking issues:

* [Restore support for pure-native/hybrid Projects](https://github.com/wix/Detox/issues/2543)
* [cannot be cast to com.facebook.react.ReactApplication](https://github.com/wix/Detox/issues/1220)
* [native android app Process crashes](https://github.com/wix/Detox/issues/1093)

:::

> However, native iOS apps are expected to work fine with Detox, and if that's your case, this introduction will help you in equal measure.

Reviewer: ```suggestion
However, native iOS apps are expected to work fine with Detox, and if that's your case, this introduction will help you just as much.
```
